### PR TITLE
Add 2 new, optional properties to educationAssignment Object

### PR DESCRIPTION
**Pull request by shravva**
Add 2 new, optional properties to educationAssignment object (http://graph.microsoft.com/beta/education/classes(id)/assignments )

addedStudentAction,
notificationChannelUrl

### CODE REVIEW

**Comment by shravva** [comment]
add tag `do not merge`

File: concepts/changelog.md [hunk 1]
```diff
@@ -30,6 +30,11 @@ For a summary of the value of these API changes, as well as recent tools, compon
 | :-------------- | :------------ | :--------------------------------------- |
 | Addition | beta | Added the [custodian](/graph/api/resources/custodian?view=graph-rest-beta&preserve-view=true), [unifiedGroupSource](/graph/api/resources/unifiedgroupsource?view=graph-rest-beta&preserve-view=true), and [siteSource](/graph/api/resources/sitesource?view=graph-rest-beta&preserve-view=true) resources and associated methods. |
 
+### Education
+| **Change type** | **Version**   | **Description**                          |
+| :-------------- | :------------ | :--------------------------------------- |
+| Addition | beta | Added new optional, properties **addedStudentAction**, **notificationChannelUrl** to [educationAssignment](/graph/api/resources/educationAssignment?view=graph-rest-beta&preserve-view=true).|
```
Comment (Lauragra) [inline concepts/changelog.md:36]: Please remove the period after optional. Please also add an entry to the EducationAssignment JSON file for this change.

**Review by Lauragra — CHANGES_REQUESTED** [review]
Please see comment.

File: concepts/changelog.md (hunk 1 above)
Comment (shravva) [reply concepts/changelog.md:36]: >Please remove the period after optional

Do you mean `,` after an optional word or `.` at the end of the sentence?

>Please also add an entry to the EducationAssignment JSON file for this change.

Already mentioned the link `[educationAssignment](/graph/api/resources/educationAssignment?view=graph-rest-beta&preserve-view=true)` where JSON repesentation provided with these new fields too.

Comment (Lauragra) [reply concepts/changelog.md:36]: In addition to the changelog.md file, we need a changelist item in this JSON file: https://github.com/microsoftgraph/microsoft-graph-docs/blob/master/changelog/Microsoft.EducationAssignment.json.

Comment (shravva) [reply concepts/changelog.md]: Updated changelog and also JSON file.

**Comment by Lauragra** [comment]
@shravva , are you ready to merge this PR?

**Review by Lauragra — APPROVED** [review]

**Comment by shravva** [comment]
Yes, its ready for merge.

**Comment by shravva** [comment]
> @shravva , are you ready to merge this PR?

Yes, it's ready for merging. May I know when it could deploy then?

**Pull request merged**